crates/testing/tests/tests_1/vote_dependency_handle.rs

```diff
@@ -99,6 +94,7 @@ async fn test_vote_dependency_handle() {
         let mut event_receiver = handle.internal_event_stream_receiver_known_impl();
         let view_number = ViewNumber::new(node_id);
 
+        #[allow(unused)]
```

Comment:
This is intentional, we make the dependency task, but it does not implement the typical `handle` method that other tasks have, so it just has to kind of sit here and read from the stream.